spec/models/history_spec.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe History, type: :model do
```

Comment:
What are your thoughts on happy-path and unhappy path testing? I notice these check for invalid cases vs valid ones.

Reply:
Hi Yes - in this case, I test for a history creation should produce invalid results since it depends on a task.